content/guides/translation-connection/step3-how-to-build-your-connector.md

```diff
@@ -0,0 +1,218 @@
+
+First, in order not to confuse these new product attributes with some of her other product attributes, it is necessary for Julia to create **a specific attribute group** named `Translations`.
+
+To create this attribute group:
+1. Go to `Settings > Attribute groups`
+2. Click on `CREATE` in the top-right corner
+3. Add a new attribute group with the code and label: `Translations`.
+
+
+### Create translation attributes
+
+Then Julia must create 6 new attributes in this attribute group:
+
+* `Translation submission name`: This attribute allows Julia to set a name to her translation project. It also allows her to easily find this project with this name in the interface of your online translation solution. Set up this attribute as followed:
+  - Attribute type: text
+  - Attribute code: `translationName`
+  - Attribute label: `Translation submission name`
+  - Attribute group: `Translations`
+  - Usable in grid: enabled
+* `Translation submitter`: This simple select attribute should contain in its options, all possible contributor names. This attribute allows Julia to associate her name to her translation project. Set up this attribute as followed:
+  - Attribute type: simple select  
+  - Attribute code: `translationSubmitter`
+  - Attribute label: `Translation submitter`
+  - Attribute group: `Translations`
+  - Usable in grid: enabled
+* `Translation queued`: This attribute allows Julia to set if a product need to be translated. Set up this attribute as followed:
+  - Attribute type: Yes/No (Boolean)
+  - Attribute code: `translationQueued`
+  - Attribute label: `Translation queued`
+  - Attribute group: `Translations`
+  - Usable in grid: enabled
+* `Translation locales`: This multi select attribute should contains in its `options` all destination locales Julia can use to make her translation requests. These same `locales` must of course also already exist in Akeneo PIM and in your online Translation tool. This attribute allows Julia to choose one or more target locales for her translation project. Set up this attribute as followed:
+  - Attribute type: multi select
+  - Attribute code: `translationLocales`
+  - Attribute label: `Translation locales`
+  - Attribute group: `Translations`
+  - Usable in grid: enabled
+* `Translation due date`: This attribute allows Julia to indicate to translators when she would like her products to be translated. Set up this attribute as followed:
+  - Attribute type: date
+  - Attribute code: `translationDate`
+  - Attribute label: `Translation due date`
+  - Attribute group: `Translations`
+  - Usable in grid: enabled  
+* `Translation status`: This attribute allows your translator to give a status about the translation project to Julia. As this last attribute has a `Read only` property, it can't be modified by Julia. Only your connector can modify this status through API and set the translation project status for each desired locale. This attribute contains these options:    
```

Comment:
```suggestion
* `Translation status`: This attribute allows your translator to give a status about the translation project to Julia. As this attribute has a `Read only` property, it can't be modified by Julia. This status can only be modified by your connector through the API and set the translation project status for each desired locale. This attribute contains these options:    
```